src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/ErrorNotificationSender.java

```diff
@@ -33,6 +41,14 @@ public ErrorNotificationSender(
         this.containerMappings = containerMappings;
     }
 
+    /**
+     * Sends an error notification to the error notification queue.
+     * @param zipFilename The zip file name
+     * @param containerName The container name
+     * @param eventId The event ID
+     * @param errorCode The error code
+     * @param cause The cause of the error
```

Comment:
Everything is in method signature order apart from the EnvelopeRejectionException cause param

Reply:
fixed